fedora/jellyfin-web.spec

```diff
@@ -30,6 +30,10 @@ Jellyfin is a free software media system that puts you in control of managing an
 %build
 
 %install
+%if 0%{?rhel} > 0 && 0%{?rhel} < 8
+# Required for CentOS build
+chown root:root -R .
```

Comment:
This is still going to break any properly built RPM build system.

You cannot do `root` things in an RPM scriptlet and SHOULD NOT build RPMs as root.

I am working on fixing all of these issues in another PR.

Reply:
Brian, I'm going to be as blunt as I can, because I think that's the only way to get this point across clearly at this point.  We do not care what you say we can and cannot do, or what's "proper".  We care what works.  And given that this has been broken for well over a month due to your previous "improvements", we're going to go with what works.  Despite pointing it out to you and asking repeatedly for you to fix it, you've argued about it and denied that it was a problem.  This whole situation has (again) prompted discussions about removing official support for rpm systems since none of us use them and it's brought the whole build system to it's knees.

TLDR: Until you're willing to put in the work and hand us something that is _guaranteed_ to work properly and not over complicate things more than they already are, stop criticizing.  Your previous efforts are leaving something to be desired